src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf

```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Et gouf keen temporären Dossier an der php.ini konfiguréiert.</target>
+                <target>Et gouf keen temporären Dossier an der php.ini konfiguréiert oder den temporären Dossier existéiert net.</target>
```

Comment:
Are you sure we should merge this? This is not a pure translation, but adds a hint that might not be correct as it doesn't exist in the source message.

Reply:
https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Validator/Resources/translations/validators.en.xlf#L195 :+1: